enterprise/server/scheduling/scheduler_client/scheduler_client.go

```diff
@@ -127,11 +133,15 @@ func (r *Registration) processWorkStream(ctx context.Context, stream scpb.Schedu
 		}
 		return true, nil
 	case msg := <-schedulerMsgs:
+		if moreWorkResponse := msg.GetAskForMoreWorkResponse(); moreWorkResponse != nil {
```

Comment:
I made a change so that the executor will request work if cpu + mem capacity are < .20 (flag configurable). Can chat offline if you don't like this default, lmk.